subsys/bluetooth/audio/vcs.c

```diff
@@ -171,7 +171,7 @@ static ssize_t write_vcs_control(struct bt_conn *conn,
 		volume_change = true;
 		break;
 	case BT_VCS_OPCODE_SET_ABS_VOL:
-		BT_DBG("Set Absolute Volume (0x%x) %u",
+		BT_DBG("Set Absolute Volume (0x%x): Current volume %u",
```

Comment:
The message here might be misleading, as `Current volume` might be interpreted as the resulting (requested) volume.
I would suggest to be more precise:  `BT_DBG("Set Absolute Volume (0x%x): %u->%u", opcode, vcs_inst.srv.state.volume, cp_val->volume` or sth like that

Reply:
I don't disagree, but at this point we do have not decoded the volume requested by the peer. 

An alternative would be to simply omit the current volume in this (we don't log that for the other volume change opcodes...) and then actually log the volume changes as `%u -> %u` as you suggested. 

Would you like me to update the PR to do that?